doc/thirdPartyRunPlugins.md

```diff
@@ -0,0 +1,35 @@
+# Third-Party Run Plugins
+
+Collection of third-party Run plugins created by the community that aren't distributed with PowerToys.
+
+## Installing / Upgrading
+
+The recommended path for the plugins is `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`.
+
+- Close PowerToys
+- Copy the plugin folder to `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`
+- Delete previous version in case of upgrade
+- Open PowerToys
+
+Refer to plugin instruction for more details.
+
+## Disclaimer
+
+The following plugins are created and maintained by third-party and not officially supported or recognized by Microsoft.  
+Users install and use these plugins at their own risk, and Microsoft takes no responsibility for their functionality or any potential issues they may cause.  
+For assistance, please contact the respective plugin developers directly.
+
+## Plugins
+
+| Plugin | Description |
```

Comment:
Let's have a language column. Some third-party plugins are in non-english language.

Reply:
I don't think this is a good idea to bring too many plugin details in PT repo. These may change and become outdated.

Reply:
Lets see where stuff goes.  Start smaller and see where it goes.  I'm happy with the current approach

Reply:
> These may change and become outdated.

valid point.